lms/djangoapps/certificates/tasks.py

```diff
@@ -18,11 +19,16 @@ class _BaseCertificateTask(PersistOnFailureTask, LoggedTask):  # pylint: disable
     abstract = True
 
 
-@task(base=_BaseCertificateTask)
-def generate_certificate(**kwargs):
+@task(base=_BaseCertificateTask, bind=True, default_retry_delay=30, max_retries=2)
+def generate_certificate(self, **kwargs):
     """
     Generates a certificate for a single user.
     """
     student = User.objects.get(id=kwargs.pop('student'))
     course_key = CourseKey.from_string(kwargs.pop('course_key'))
+    expected_verification_status = kwargs.pop('verification', None)
```

Comment:
You're passing this kwarg in as `'expected_verification_status'`.  Also, I would add a note in the docstring for this function about this kwarg and why we need it (and maybe a short sentence for `student` and `course_key` as well).

Reply:
Oh, oops. Thanks